assets/src/infrastructure/ui/layout/modal/modalCloseButtonProps/index.tsx

```diff
@@ -6,9 +6,8 @@ import { Close } from '@appDisplay/icons';
 import './styles.scss';
 
 const modalCloseButtonProps: ButtonProps = {
-	className: 'ee-confirm-close',
+	className: 'ee-icon-button ee-confirm-close',
 	icon: Close,
-	variant: 'unstyled',
```

Comment:
`unstyled` is a bit tricky since it also adds some styles which have conflicted with border styles: https://github.com/chakra-ui/chakra-ui/blob/master/packages/chakra-ui/src/Button/styles.js#L196